tests/test_ann.py

```diff
@@ -161,6 +162,8 @@ def test_construction(self):
             ),
         ]
 
+        color = CIELabColor(40.0, 50.0, 6.0)
```

Comment:
Maybe add a comment what that color looks like, just so that the users know what to expect?